.github/workflows/modrinth-publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Automatically publish to Modrinth and CurseForge
+on:
+  release:
+    types: [published]
+jobs:
+  publish-to-modrinth:
+    runs-on: ubuntu-latest
+    steps:
+       - name: Get the version
+         id: get_version
+         run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
```

Comment:
May not work because on github versions I use _vx.y.z_ but filenames don't have that "v"

Reply:
Should be fixed in the latest commit